src/apps/TCO2_Dashboard/content_Toucan.py

```diff
@@ -0,0 +1,202 @@
+import dash_bootstrap_components as dbc
+from dash import html
+from dash import dcc
+from helpers import (drop_duplicates, date_manipulations, black_list_manipulations,
+                     region_manipulations, subsets)
+from Figures import *
+from data_related_constants import rename_map, retires_rename_map
+# from get_data import execute
+from import_data import get_data
+
+df, df_retired = get_data()
```

Comment:
So I believe the way you're calling `get_data` and `get_data_pool` is going to result in the data being loaded once when the application is deployed and then never refreshed

This forum post walks through the different options: https://community.plotly.com/t/how-to-refresh-data-on-fly/13069

Ultimately I think using an `Interval` components that periodically fetches the new data and updates all the figures on the server-side is what we want, but implementing that will be somewhat involved

For now I think a simpler approach would be just caching the dataset for one day using `flask_caching` as shown in the second example here: https://dash.plotly.com/performance#memoization

We can walk through that together if you want